src/manage/project-details.md

```diff
@@ -12,33 +12,44 @@ Great tools to inspect the project history and compare changes between project v
 - use [Create diff](./plugin/#create-diff-processing-toolbox) to extract changes made between two versions of a layer
 :::
 
-## Project history
+## Project history and advanced changelog
 To see the project history:
-1. Log in <AppDomainNameLink /> and open your project
-2. Navigate to the **HISTORY** tab
-3. Here, you can find:
-  - the **version** of the project based on the number of synchronisations. Click on the version to see the [changelog](#advanced-changelog).
-  - the time when was the version **created**
-  - the **Author** of this version (who made the changes)
-  - overview of the number of files that were **added, deleted and updated**
-  - **Size** of the project
-  - option to **download** this version of the project
-  
-![Mergin Maps project history web](./web-history-2.jpg "Mergin Maps project history")
+1. Log in <AppDomainNameLink /> and go to your project
+2. Navigate to the **History** tab. 
 
-:::tip
-Our blog [Downloading a previous version of your project](https://merginmaps.com/blog/support-tip-downloading-a-previous-version-of-your-project) contains detailed steps on how to download projects that are too large to be downloaded from the <DashboardShortLink />.
-:::
+   Here you can find all available versions of the project. These versions can be downloaded to your computer which can be helpful if you need to create a backup or [restore previous version](#how-to-revert-to-an-older-version) of the project.
+   
+   ![Mergin Maps project history web](./dashboard-projects-history.jpg "Mergin Maps project history")
+
+3. Click on a version to see the more details about what was changed.
+   
+   You can inspect the list of **added**, **edited** and **removed** files.
+   
+   ![Overview of changes made in a project in the dashboard](./dashboard-history-details.jpg "Overview of changes made in a project in the dashboard")
+
+4. For GeoPackage layers, there is an option to see even more. Click on **Show advanced** 
+
+   ![Show advanced changelog of a layer](./dashboard-history-show-advanced-changelog.jpg "Show advanced changelog of a layer")
+
+5. Now you can see the overview of feature changes in a specific layer across all attributes.
+   
+   Here, there are multiple changes in the `trees` layer. One feature (`fid: 9`) was added, one feature (`fid: 6`) was deleted. There are two modified features: `fid: 2` was moved to a new position and in `fid: 8`, the value of attribute `species` was changed.
```

Comment:
Edit last sentence to be a list:
There are two modified features: 
- `fid: 2` was moved to a new position 
- For `fid: 8`, the value of attribute `species` was changed